test/test_sched.rs

```diff
@@ -0,0 +1,36 @@
+
+#[cfg(any(target_os = "android",
```

Comment:
You should move the `#[cfg()]` into test/test.rs, for consistency with other files.